src/components/RadioButtonList/RadioButtonOption.tsx

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2019 - 2023 Gemeente Amsterdam
+
+import { Fragment } from 'react'
+
+import FormField from 'components/FormField'
+import RadioButtonComponent from 'components/RadioButton'
+import TextArea from 'components/TextArea'
+
+import type { RadioButtonListProps } from './RadioButtonList'
+import { StyledLabel } from './styled'
+import type { RadioButtonOption as RadioButtonOptionType } from './types'
+import TopicLabel from '../TopicLabel'
+
+interface Props {
+  defaultValue?: string
+  formValidation: RadioButtonListProps['formValidation']
+  groupName: RadioButtonListProps['groupName']
+  index: number
+  option: RadioButtonOptionType
+  onChange: RadioButtonListProps['onChange']
+  radioOptions: RadioButtonListProps['options']
+}
+
+export const RadioButtonOption = ({
+  defaultValue,
+  formValidation,
+  groupName,
+  index,
+  option,
+  onChange,
+  radioOptions,
+}: Props) => {
+  let component = (
+    <Fragment key={option.key || option.name}>
```

Comment:
why one of both?

Reply:
seems key is always there right?

Reply:
Good point. Seems like it yes. Didn;t change the old code though. But will fix it.

Reply:
Never mind, it can be an empty string. Then you want to take the name property.

<img width="301" alt="Screenshot 2023-08-02 at 09 53 10" src="https://github.com/Amsterdam/signals-frontend/assets/46756331/3b8e2aa4-ee44-4962-89af-4eb9b96da150">